src/frontend/apps/desk/src/features/members/components/MemberGrid.tsx

```diff
@@ -127,7 +128,10 @@ export const MemberGrid = ({ teamId, currentRole }: MemberGridProps) => {
         />
       </Card>
       {isModalMemberOpen && (
-        <ModalAddMembers onClose={() => setIsModalMemberOpen(false)} />
+        <ModalAddMembers
+          onClose={() => setIsModalMemberOpen(false)}
+          team={team}
```

Comment:
I am not sure that it's nitpicking. The code is functional, so actually this comment might be. I think we start facing some props drilling. The team data is passed from the `[id].tsx` page down to the select component, over few levels of nesting. Could we take the benefits of React query, and use it as a global store for server-side fetched data?

Reply:
I am totally okay with refactoring it later on after we ship a first PoC